lib/tools/broccoli/node-modules-dest-copy.ts

```diff
@@ -69,7 +70,7 @@ export class DestCopy implements IBroccoliPlugin {
 				});
 			}
 		});
-		if (!_.isEmpty(this.dependencies)) {
+		if (!_.isEmpty(this.dependencies) && !this.$config.debugLivesync) {
```

Comment:
Please note that I saw the CLI code a month ago for the first time. So, I am just scratching the surface.  I think we both have relatively the same knowledge about it. I am just trying to fix some of its issues that annoy our customers and our internal teams. This has the greatest priority for me. So I count on your assistance, good will and support.